tools/cli/go-whisk-cli/commands/api.go

```diff
@@ -331,57 +332,68 @@ var apiListCmd = &cobra.Command{
     PreRunE:       setupClientConfig,
     RunE: func(cmd *cobra.Command, args []string) error {
         var err error
+        var retApiList *whisk.ApiListResponse
+        var retApi *whisk.ApiGetResponse
         var retApiArray *whisk.RetApiArray
 
         if whiskErr := checkArgs(args, 0, 3, "Api list",
             wski18n.T("Optional parameters are: API base path (or API name), API relative path and operation.")); whiskErr != nil {
             return whiskErr
         }
 
-        api := new(whisk.Api)
-        api.Namespace = client.Config.Namespace
+        // Get API request body
+        apiGetReq := new(whisk.ApiGetRequest)
+        apiGetReq.Namespace = client.Config.Namespace
+
+        // Get API request options
+        apiGetReqOptions := new(whisk.ApiGetRequestOptions)
 
-        options := new(whisk.ApiListOptions)
-        options.Limit = flags.common.limit
-        options.Skip = flags.common.skip
+        // List API request query parameters
+        apiListReqOptions := new(whisk.ApiListRequestOptions)
+        apiListReqOptions.Limit = flags.common.limit
+        apiListReqOptions.Skip = flags.common.skip
 
         if (len(args) == 0) {
-            retApiArray, _, err = client.Apis.List(options)
+            retApiList, _, err = client.Apis.List(apiListReqOptions)
             if err != nil {
-                whisk.Debug(whisk.DbgError, "client.Apis.List(%s) error: %s\n", options, err)
+                whisk.Debug(whisk.DbgError, "client.Apis.List(%#v) error: %s\n", apiListReqOptions, err)
                 errMsg := wski18n.T("Unable to get API: {{.err}}", map[string]interface{}{"err": err})
```

Comment:
:+1:  re: use of `list` in message

See above comment re: API variations that make it complex to display the entire API name context.